### PR TITLE
FIX:TD-fixed baloon cursor hotspot to be on arrow tip

### PR DESCRIPTION
**Pull request by senjabl**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/TechDraw/Gui/QGIViewBalloon.cpp [hunk 1]
```diff
@@ -173,8 +173,8 @@ void QGIViewBalloon::onAttachPointPicked(QGIView *view, QPointF pos)
 
     if (balloon->OriginIsSet.getValue() == false) {
         /* Move origin by half of cursor size */
```
Comment (fjullien) [inline src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: Could you remove this comment ?
BTW, your solution is much cleaner. Thanks.

Comment (senjabl) [reply src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: OK, removed....

Comment (fjullien) [reply src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: I might be a bit picky here but you should have squashed this commit with you previous one.

Comment (senjabl) [reply src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: What do you mean? If I go to "Files changed" on the top of this page, everything is there. Am I missing something?

**Comment by WandererFan** [comment]
merged.

File: src/Mod/TechDraw/Gui/QGIViewBalloon.cpp (hunk 1 above)
Comment (fjullien) [reply src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: In your local git branch you can merge your two commits (rebase -i). Then you force push the resulting commit. Your pull request will be updated automatically.

Comment (senjabl) [reply src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: Ok, WandererFan already merged it to master, but I will remember it for next time.
Git is very confusing....

Comment (fjullien) [reply src/Mod/TechDraw/Gui/QGIViewBalloon.cpp]: No problem.

**Pull request merged**